.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will build a .NET project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-net
+
+name: .NET
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
Need Dev branch as well as in ticket description

Reply:
> Need Dev branch as well as in ticket description

fixed